{{ cookiecutter.project_slug }}/README.uv.jinja

```diff
@@ -0,0 +1,87 @@
+This is a Python application that uses [uv] for packaging and dependency management.
+It  provides [`pre-commit`][pre-commit] hooks for various linters
+and formatters and automated tests using [`pytest`][pytest] and [GitHub Actions].
+Pre-commit hooks are automatically kept updated with a dedicated GitHub Action, this can
+be removed and replaced with [pre-commit.ci] if using a public repo.
+
+It was developed by the [Imperial College Research Software Engineering Team].
+
+## Usage
+
+To get started:
+
+1. [Download and install uv] following the instructions for your OS.
+2. Set up the virtual environment:
+
+   ```bash
+   uv venv .venv
+   ```
+
+3. Initialise the Git repository:
+
+   ```bash
+   git init
+   ```
+
+4. Activate the virtual environment, or just preface your commands with `uv run` to use
+the virtual environment (see [uv activate] for more info). Each time `uv run` is called,
+the virtual environment will install any missing dependencies automatically:
+
+   ```bash
+   source .venv/bin/activate
+   uv sync --all-groups
+   ```
+
+   or
+
+   ```bash
+   uv run <command>
+   ```
+
+5. Install the pre-commit git hooks:
+
+   ```bash
+   uv run pre-commit install
+   ```
+
+6. Update the pre-commit hooks
+
+   ```bash
+   uv run pre-commit autoupdate
+   ```
+
+7. Run the main app:
+
+   ```bash
+   uv run python -m {{ cookiecutter.project_slug }}
+   ```
+
+8. Run the tests:
+
+   ```bash
+   uv run pytest
+   ```
+
+9. Create an initial commit (it's possible there might be some failures in pre-commit):
+
+   ```bash
+   git add .
+   git commit -m "Initial commit"
+   ```
+
+## Updating Dependencies
+
+Use the commands `uv add <package>` and `uv remove <package>` to add or remove dependencies.
+
+- Include the optional flag `--group` to add/remove dependencies from specific groups.
```

Comment:
I'd suggest using `--dev` for development dependencies and `--group <group-name>` for all others. Maybe make these two separate dot points.